surface/src/3rdparty/opennurbs/opennurbs_bezier.cpp

```diff
@@ -2320,7 +2320,6 @@ bool ON_BezierSurface::GetNurbForm( ON_NurbsSurface& n ) const
     n.m_knot[1][m_order[1]-2] = 0.0;	//m_domain[1].Min();
     n.m_knot[1][m_order[1]-1] = 1.0;	//m_domain[1].Max();
     rc = ON_ClampKnotVector( n.m_order[0], n.m_cv_count[0], n.m_knot[0], 2 );
-    rc = ON_ClampKnotVector( n.m_order[1], n.m_cv_count[1], n.m_knot[1], 2 );
```

Comment:
Removing the previous line would make more sense though this code might point to a potential bug